docs/user/reference/cloud/list-of-supported-clouds/the-lxd-cloud-and-juju.md

```diff
@@ -61,7 +61,10 @@ Attributes:
 
 #### `interactive`
 Attributes:
-- trust-password: the LXD server trust password (required)
+> Added in Juju 3.6.4.
```

Comment:
This note only applies to trust-token, right? I'd move it right after trust-token, in parentheses.